src/components/BenefitPlanGroupBeneficiariesSearcher.js

```diff
@@ -63,8 +64,25 @@ function BenefitPlanGroupBeneficiariesSearcher({
   + `${modulesManager.getRef('socialProtection.route.benefitPackage')}`
     + `/group/${groupBeneficiary?.id}`);
 
+  const addUpdatedGroupBeneficiary = (groupBeneficiary, status) => {
```

Comment:
const addUpdatedGroupBeneficiary and addUpdatedBeneficiary looks really simillar in case of implementation. Can we create one const function for that? @jdolkowski